integration/test_auth.py

```diff
@@ -420,38 +417,54 @@ def test_import_users_with_password(api_key):
 
 def test_password_reset(new_user_email_unverified, api_key):
     link = auth.generate_password_reset_link(new_user_email_unverified.email)
-    oob_code = _validate_link_url(link, check_continue_url=False)
-    assert new_user_email_unverified.email == _reset_password(oob_code, "newPassword", api_key)
+    assert isinstance(link, six.string_types)
+    query_dict = _extract_link_params(link)
+    user_email = _reset_password(query_dict['oobCode'], "newPassword", api_key)
```

Comment:
Sorry I didn't notice this earlier. But please use single quotes for string literals: `'newPassword'`

Reply:
done.